packages/api/src/schema/types/order/pick-up-location.ts

```diff
@@ -7,5 +7,11 @@ export default [
       address: Address
       geoPoint: GeoPosition
     }
+    type PickUpLocation {
```

Comment:
I think it's not needed to have an Order field on the OrderPickUpLocation so please remove the old type and re-use PickUpLocation in OrderDeliveryPickUp too. Aaand, i think pickUpLocations on OrderDeliveryPickUp should be marked deprecated in the graphql schema.